src/naksu/boxversion/boxversion.go

```diff
@@ -125,51 +132,16 @@ func GetVagrantBoxAvailVersionDetails() (string, string, error) {
 }
 
 // reallyGetVagrantBoxAvailVersionDetails returns version string (e.g. "digabi/ktp-qa") and
-// version number (e.g. "69") by getting Vagrantfile -> metadata.json
+// version string (e.g. "SERVER7108X") by getting Vagrantfile from the AbittiVagrantURL
```

Comment:
Kommentissa toisto: "returns **version string** (e.g. "digabi/ktp-qa") and **version string** (e.g. "SERVER7108X")".

Reply:
:+1: